src/app/workbench/WorkbenchViews.tsx

```diff
@@ -155,7 +113,26 @@ export function WorkbenchViews({ index, type }: IWorkbenchViewsProps) {
           </div>
         ) : null}
         <div ref={setRef} className="d-flex flex-grow-1">
-          {wb}
+          <Mosaic<string>
+            renderTile={(id, path) => {
+              const currView = views.find((v) => v.uniqueId === id);
+              if (currView) {
+                return (
+                  <WorkbenchView
+                    removeCallback={removeCallback}
+                    dragMode={mosaicDrag}
```

Comment:
```suggestion
                    mosaicDrag={mosaicDrag}
```

I think we should keep the convention that the variable is matching the state setter. I would rename `dragMode` everywhere and even place the lines next to each other to make the connection clearer. This applies also to the other files.

Reply:
Agreed. I realized the "onDragStart" and end functions from mosaic are related to moving windows around, not resizing them, so i'm no longer passing the setMosaicState function down. But renamed the dragMode -> mosaicDrag still.

Reply:
Thanks.